cluster/images/etcd-empty-dir-cleanup/Dockerfile

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gliderlabs/alpine
+FROM busybox
+
+COPY etcdctl etcd-empty-dir-cleanup.sh /
+RUN chmod a+rx /etcdctl /etcd-empty-dir-cleanup.sh
 
-RUN apk-install bash
-ADD etcd-empty-dir-cleanup.sh etcd-empty-dir-cleanup.sh
-ADD etcdctl etcdctl
 ENV ETCDCTL /etcdctl
 ENV SLEEP_SECOND 3600
-RUN chmod +x etcd-empty-dir-cleanup.sh
-CMD bash /etcd-empty-dir-cleanup.sh
+
+# nobody:nobody
+USER 65534:65534
```

Comment:
I'm surprised this works... I guess etcd doesn't need any credentials? Did you verify that the empty directories actually are cleanup up?